src/lib.rs

```diff
@@ -343,7 +345,8 @@ mod test;
 #[cfg(feature = "tracing-unstable")]
 mod trace;
```

Comment:
I don't think there's anything here that needs to be exported publicly/documented but let me know if that's wrong, @kmahar.

Reply:
the only thing explicitly in the API around tracing is the `tracing_max_document_length_bytes` option in `ClientOptions`. that already has the docsrs annotations and appears to show up correctly in the docs: https://docs.rs/mongodb/2.4.0-beta.1/mongodb/options/struct.ClientOptions.html#structfield.tracing_max_document_length_bytes